### PR TITLE
[FIX] spreadsheet_dashboard: fix lost margins on dropdowns

### PR DESCRIPTION
**Pull request by edi-odoo**
This PR fixes the missing margins on either side of the dropdown buttons in the control panel.

Issue in the pad: 
[EDI] Dashboards: spacing issue between inputs https://tinyurl.com/2gzrpftg

The CSS for the .o-filter-value wasn't targeting anything. This was fixed by moving the CSS into its correct parent.
Note the whole layout for Dashboard needs reworking so I didn't make any further changes just yet.

task-3326566
part of task-3326263

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by stefanorigano** [comment]
Thanks @edi-odoo , but the branch is red and runbot won't load. 
Rebase issue? Could you have a look, thanks!

**Comment by edi-odoo** [comment]
> Thanks @edi-odoo , but the branch is red and runbot won't load. Rebase issue? Could you have a look, thanks!

Looks fine on my end https://runbot.odoo.com/runbot/batch/1050111

**Review by stefanorigano — CHANGES_REQUESTED** [review]

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.scss [hunk 1]
```diff
@@ -18,7 +18,6 @@
             background-color: $o-gray-100;
         }
     }
-
```
Comment (stefanorigano) [inline addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.scss:21]: unrelated unnecessary diff ;)

File: addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.scss [hunk 2]
```diff
@@ -73,7 +74,7 @@
             .o_field_many2manytags,
             .date_filter_values {
                 display: flex;
-                gap: 3px;
+                gap: 8px;
```
Comment (stefanorigano) [inline addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.scss:77]: `map-get($spacers, 2);` ?

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Hello @edi-odoo , two minor remarks, thanks 👍

**Comment by edi-odoo** [comment]
Hi @odoo/rd-spreadsheet , could you check this please ? Thanks !

**Comment by LucasLefevre** [comment]
LGTM :+1: 
Thanks
Could you rebase the enterprise branch to a saas-16.3 commit ? (or delete the branch I believe it would also work)
The ci/codeowner and ci/style checks are failing because the enterprise branch is de-synchronized and they are trying to check too many commits.

**Comment by edi-odoo** [comment]
> or delete the branch I believe it would also work

Thanks @LucasLefevre , I deleted the enterprise branch ;)

**Comment by LucasLefevre** [comment]
robodoo r+
